modules/example/src/smithy4s/example/FooService.scala

```diff
@@ -5,7 +5,7 @@ import smithy4s.syntax._
 trait FooServiceGen[F[_, _, _, _, _]] {
   self =>
 
-  def getFoo() : F[Unit, Nothing, GetFooOutput, Nothing, Nothing]
+  def getFoo() : F[PrimitiveType(Unit),Ref(smithy4s.example,GetFooOutput),List()]
```

Comment:
Something is obviously off there, the `toString` is being called on `Type` somewhere it shouldn't be

Reply:
yeah forgot to checkin .